### PR TITLE
Fixes to maps & macros

### PR DESCRIPTION
**Pull request by lecross**

### CODE REVIEW

File: content/map/6/3/readme.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ es:
     reducirían notablemente el tiempo necesario para una intervención médica
     efectiva. La página web de “Enfermesoma” ofrece un explorador de relaciones
     interactivas entre enfermedades y enlaces a documentos y bases de datos en
-    línea en http://diseasome.eu.
+    línea en ![http://diseasome.eu](http://diseasome.eu).
```
Comment (bherr2) [inline content/map/6/3/readme.md:71]: Pretty sure adding a ! makes it an image...

Comment (bherr2) [reply content/map/6/3/readme.md:71]: @lecross

Comment (lecross) [reply content/map/6/3/readme.md:71]: That was the only trick I found for making a url into text. If there is another method, let me know. I'll check with Lisel about removing the link entirely.

Comment (bherr2) [reply content/map/6/3/readme.md:71]: Use `<http://diseasome.eu>`

**Review by bherr2 — APPROVED** [review]

**Pull request merged**